heron/executor/src/python/heron_executor.py

```diff
@@ -976,6 +979,14 @@ def get_commands_to_run(self):
     # During shutdown the watch might get triggered with the empty packing plan
     if len(self.packing_plan.container_plans) == 0:
       return {}
+    if self._get_instance_plans(self.packing_plan, self.shard) is None and self.shard != 0:
+      retval = {}
+      retval['heron-shell'] = [
+        '%s' % self.heron_shell_binary,
```

Comment:
why in this case a `heron-shell` command is returned?

Reply:
health check in shell determines if aurora returns